holidays/countries/united_arab_emirates.py

```diff
@@ -50,40 +55,38 @@ def _populate(self, year):
         super()._populate(year)
 
         # New Year's Day.
-        self._add_new_years_day("New Year's Day")
+        self._add_new_years_day(tr("رأس السنة الميلادية"))
 
         # Commemoration Day.
         if year >= 2015:
-            self._add_holiday("Commemoration Day", *((DEC, 1) if year >= 2019 else (NOV, 30)))
+            self._add_holiday(tr("يوم الشهيد"), *((DEC, 1) if year >= 2019 else (NOV, 30)))
 
         # National Day.
-        self._add_holiday("National Day", DEC, 2)
-        self._add_holiday("National Day Holiday", DEC, 3)
+        self._add_holiday(tr("اليوم الوطني"), DEC, 2)
+        self._add_holiday(tr("اليوم الوطني"), DEC, 3)
 
         # Eid al-Fitr.
-        eid_al_fitr = "Eid al-Fitr"
-        self._add_eid_al_fitr_day(eid_al_fitr)
-        self._add_eid_al_fitr_day_two(f"{eid_al_fitr} Holiday")
-        self._add_eid_al_fitr_day_three(f"{eid_al_fitr} Holiday")
+        self._add_eid_al_fitr_day(tr("عيد الفطر"))
+        self._add_eid_al_fitr_day_two(tr("عطلة عيد الفطر"))
+        self._add_eid_al_fitr_day_three(tr("عطلة عيد الفطر"))
 
         # Arafat Day & Eid al-Adha.
-        self._add_arafah_day("Arafat (Hajj) Day")
+        self._add_arafah_day(tr("وقفة عرفة"))
 
-        eid_al_adha = "Eid al-Adha"
-        self._add_eid_al_adha_day(eid_al_adha)
-        self._add_eid_al_adha_day_two(f"{eid_al_adha} Holiday")
-        self._add_eid_al_adha_day_three(f"{eid_al_adha} Holiday")
+        self._add_eid_al_adha_day(tr("عيد الأضحى"))
```

Comment:
```suggestion
        # Eid al-Adha.
        self._add_eid_al_adha_day(tr("عيد الأضحى"))
```

Could you also add a comment for the 2 following lines?